tests/anmykh/test_is_string.py

```diff
@@ -0,0 +1,2 @@
+def test_is_string():
+    assert isinstance("sad", str)
```

Comment:
no need to make 1 file for 1 test,   

group tests together , if they relate to same idea e.g.  (errors, conversion etc.)

Reply:
fixed